src/component/rent/RentModalUserList.tsx

```diff
@@ -8,24 +10,38 @@ type Props = {
 };
 
 const UserList = ({ user, setSelectedUser, closeModal }: Props) => {
+
+  const librarian = useRecoilValue(userAtom);
```

Comment:
```suggestion
  const currentUser= useRecoilValue(userAtom);
```
`librarian` 는 사서유저를 뜻하는데, 맥락상 사서유저보다는 현재유저가 더 적합한거 같습니다. 

`librarian`으로 하는 특별한 이유가 있나요?!